src/containers/Tenant/Diagnostics/TopQueries/getTopQueriesColumns.tsx

```diff
@@ -66,6 +86,20 @@ const userSIDColumn: Column<KeyValueRow> = {
     align: DataTable.LEFT,
 };
 
+const topQueriesQueryTextColumn: Column<KeyValueRow> = {
+    name: TOP_QUERIES_COLUMNS_IDS.TopQueriesQueryText,
+    header: tableColumnsNames[TOP_QUERIES_COLUMNS_IDS.TopQueriesQueryText],
+    render: ({row}) => <OneLineQueryWithPopover value={row.QueryText?.toString()} />,
+    sortable: false,
+};
```

Comment:
Bad naming. You start name with "topQueries", but this column isn't used in initial `TopQueries` component, but in `TenantOverview`. It's better to think of better naming, probably not scope-based, but functionality-based